class.jetpack-onboarding-end-points.php

```diff
@@ -531,12 +533,54 @@ static function install_theme() {
 					wp_send_json_error('Could not install theme (unspecified server error)');
 					die();
 				}
-			} 
+			}
 		}
 
 		wp_send_json_success( $theme_id );
 	}
 
+	static function have_contact_info_widget( $widgets ) {
+		foreach ($widgets as $widget) {
+    		if ( strpos( $widget, 'widget_widget_contact_info' ) ) {
+    			return true;
+    		}
+    	}
+    	return false;
+	}
+
+	static function add_business_address() {
+		check_ajax_referer( self::AJAX_NONCE, 'nonce' );
+		$active_sidebars = get_option( 'sidebars_widgets' );
+		$widget_options = get_option( 'widget_widget_contact_info' );
+		if( isset( $active_sidebars['sidebar-1'] ) &&
```

Comment:
Just to clarify, you could have recently created your site, but you could use a theme that registers sidebars with custom ids like `default-sidebar` or `left-sidebar`, and not the default IDs. So asserting that a `sidebar-1` sidebar exists is not reliable in any case.